src/log-receiver/log-receiver.service.ts

```diff
@@ -17,68 +16,78 @@ import { Logs } from 'src/schema/logs.schema';
  */
 @Injectable()
 export class LogReceiverService {
+
   // Issue Creator for the Log Types
   cpuUtilizationIssueCreator: CpuUtilizationIssueCreatorComponent;
   timeoutIssueCreator: TimeoutIssueCreatorComponent;
   cbOpenIssueCreator: CbOpenIssueCreatorComponent;
   errorResponseIssueCreator: ErrorResponseIssueCreatorComponent;
-
   constructor(
     private http: HttpService,
     @InjectModel('logs') private logModel: Model<Logs>,
   ) {
     // Create an Issue Creator for each LogType
-    this.cpuUtilizationIssueCreator = new CpuUtilizationIssueCreatorComponent(
-      http,
-    );
+    this.cpuUtilizationIssueCreator = new CpuUtilizationIssueCreatorComponent(http);
     this.timeoutIssueCreator = new TimeoutIssueCreatorComponent(http);
     this.cbOpenIssueCreator = new CbOpenIssueCreatorComponent(http);
-    this.errorResponseIssueCreator = new ErrorResponseIssueCreatorComponent(
-      http,
-    );
+    this.errorResponseIssueCreator = new ErrorResponseIssueCreatorComponent(http);
   }
 
   /**
    * Handling of Log messages
```

Comment:
put more info into short description to omit long description. this does not apply every where, but in this case it works, i guess.
```suggestion
   * Handles Log messages by delegating them to the respective IssueCreator
```